sqrl-planner/src/main/java/com/datasqrl/engine/log/postgres/PostgresTable.java

```diff
@@ -0,0 +1,38 @@
+package com.datasqrl.engine.log.postgres;
+
+import com.datasqrl.canonicalizer.Name;
+import com.datasqrl.config.ConnectorFactory.IConnectorFactoryContext;
+import com.datasqrl.config.TableConfig;
+import com.datasqrl.engine.log.Log;
+import com.datasqrl.io.tables.TableSink;
+import com.datasqrl.io.tables.TableSinkImpl;
+import com.datasqrl.io.tables.TableSource;
+import com.datasqrl.plan.table.RelDataTypeTableSchema;
+import java.util.List;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class PostgresTable implements Log {
+
+  String topicName;
```

Comment:
topic?

Reply:
Yes, I felt like it would be good to emphasize that even this is a table it behaves more like a topic. But changed it because I wasn't so sure as well.